GameData/ROEngines/PartConfigs/RD301_RE.cfg

```diff
@@ -0,0 +1,90 @@
+PART
+{
+	name = ROE-RD301
+	module = Part
+	author = Alcentar, capkirk
+	rescaleFactor = 0.7773
+
+	ROESetEngineDefaults = LIQUID_ENGINE
+
+	MODEL
+	{
+		model = ROEngines/Assets/RealEngines/RD120
+		scale = 0.64, 0.64, 0.64
+	}
+	scale = 0.64
+
+	node_stack_top = 0.0, 0.7122473, 0.0, 0.0, 1.0, 0.0
+	node_stack_bottom = 0.0, -0.9684762, 0.0, 0.0, -1.0, 0.0, 0
+	node_attach = 0.0, 1.116298, 0.0, 0.0, 1.0, 0.0, 4
```

Comment:
This is somewhat a weird way to do this, using all of MODEL.scale, part.scale, and part.rescaleFactor.  I certainly don't know what to expect!  Can we strip all three out [implying the default stock rescaleFactor of not-1... 0.625 I think?], set the attach node positions correctly for that, and then just correctly set rescaleFactor to where it needs to be?

Reply:
Fixed, uses rescale now.